src/MakersOfDenmark.Domain/Enums/MakerSpaceRoles.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MakersOfDenmark.Domain.Enums
+{
+    public enum MakerSpaceRoles
+    {
+        RegularUser = 0,
```

Comment:
Måske blot "User"

Reply:
En "User" er en vag betegnelse, fordi det beskriver i princippet alle.
en "RegularUser" tilbyder ekstra beskrivelse til hensigten med denne enum,
Så jeg er egentlig uenig

Reply:
Det er fair. Kan for mit vedkommende markeres som resolved med mindre julian har indvendinger